book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :title, :author
```

Comment:
- [ ] Hello, since we will in the future modify book title and author, other only setting Getters, consider adding Setters too

```suggestion
  attr_accessor :title, :author, :rentals
```